app/views/documents/form1095as/edit.html.erb

```diff
@@ -1,3 +1,9 @@
 <% content_for :form_question, "Attach your 1095-A's" %>
 
-<% content_for :form_help_text, "Based on your answers, you should have received the following document. You may need to download this from the marketplace or the exchange for your state." %>
+<% content_for :form_help_text do %>
+  <strong>You should have this form</strong>
+  because you said that you purchased health insurance from the marketplace.
```

Comment:
maybe we can link "marketplace" to https://www.irs.gov/affordable-care-act/individuals-and-families/the-health-insurance-marketplace